lib/rules/no-script-url.js

```diff
@@ -31,18 +33,26 @@ module.exports = {
 
     create(context) {
 
+        /**
+         * Check whether a node's static value starts with "javascript:" or not.
+         * And report an error for unexpected script URL.
+         * @param {ASTNode} node node to check
+         * @returns {void}
+         */
+        function check(node) {
+            const value = astUtils.getStaticStringValue(node);
+
+            if (typeof value === "string" && value.toLowerCase().indexOf("javascript:") === 0) {
+                context.report({ node, messageId: "unexpectedScriptURL" });
+            }
+        }
         return {
-
-            Literal(node) {
-                if (node.value && typeof node.value === "string") {
-                    const value = node.value.toLowerCase();
-
-                    if (value.indexOf("javascript:") === 0) {
-                        context.report({ node, messageId: "unexpectedScriptURL" });
-                    }
+            Literal: check,
+            TemplateLiteral(node) {
+                if (node.parent && node.parent.type !== "TaggedTemplateExpression") {
```

Comment:
```suggestion
                if (!(node.parent && node.parent.type === "TaggedTemplateExpression")) {
```

I think `TemplateLiteral` node should always have a parent node. But, if we are checking that to be sure, then it would be correct to include those that don't have a parent since they're not tagged templates.